portfolio/src/main/webapp/index.html

```diff
@@ -7,24 +7,54 @@
     <script src="script.js"></script>
   </head>
   <body>
+
+    <div class='parallax' id='home'>
+        <div class='dot'>
+            <p> <b> Cindy Li's <br> Portfolio </b> </p>
+        </div>
+
+        <div class='border'> </div>
+    </div>
+
+    <div class='nav'>
+        <a href='index.html#home' class='active'>Home</a>  
+        <a href='index.html#content'>About Me</a>  
+        <a href = 'projects.html'> My Projects</a> 
+    </div>
+
     <div id="content">
-      <h1 id = 'port_section'>Cindy's Portfolio</h1>
-      <p><a href = 'projects.html'> My Projects</a></p>
-      <p>I am a rising junior at Brown University, studying Applied Math - Computer Science. My academic interests lie primarily in machine learning and data science.
-          <br> In my free time, I enjoy reading, listening to music, and hanging out with my dog. I adore giant pandas and hope to one day visit the panda sanctuary in Sichuan, China (along with a million other places).
-      </p>
-      <img src = '/images/surprise-panda.jpg' alt = 'panda image' width = 500px height = 500px>
-      <p>Click here to get a random place I want to travel to:</p>
-      <button onclick="addRandomGreeting()">Wanderlust</button>
-      <div id="greeting-container"></div>
+      <br>
+      <h1 class = 'title'>About Me</h1>
+      
+      <div id = 'bio'>
+        <p>I am a rising junior at Brown University, studying Applied Math - Computer Science. My academic interests lie primarily in machine learning and data science.
+            <br> In my free time, I enjoy reading, listening to music, and hanging out with my dog. I adore giant pandas and hope to one day visit the panda sanctuary in Sichuan, China (along with a million other places).
+        </p>
+        <img src = '/images/surprise-panda.jpg' alt = 'panda image' width = 500px height = 500px>
+      </div>
+
     </div>
-    <div id = 'contact'>
-        <hr>
-        <h3> Contact Info </h3> 
-        <p> <b>Email:</b> cindy_li@brown.edu <br>
-            <b>Cell:</b> 631-406-3344 <br>
-            <b>LinkedIn:</b> <a href = 'https://www.linkedin.com/in/cindy-li-2022/'>https://www.linkedin.com/in/cindy-li-2022/</a>
-        </p> 
+
+    <div id = 'side-bar'>
+        <div id = 'button-stuff'>
+                <p>Click here to get a random place I want to travel to:</p>
+                <button onclick="addRandomGreeting()">Wanderlust</button>
```

Comment:
nit: You might consider updating this function name since it doesn't really seem like a greeting anymore.